src/proposals/20250306/SparkEthereum_20250306.t.sol

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { DataTypes } from "sparklend-v1-core/contracts/protocol/libraries/types/DataTypes.sol";
+
+import { Base }     from 'spark-address-registry/Base.sol';
+import { Ethereum } from 'spark-address-registry/Ethereum.sol';
+
+import { IMetaMorpho, MarketParams } from 'metamorpho/interfaces/IMetaMorpho.sol';
+
+import { ReserveConfig } from "src/test-harness/ProtocolV3TestBase.sol";
+
+import 'src/test-harness/SparkTestBase.sol';
```

Comment:
I guess it's a silly remark because it's the test and doesn't affect how it works. But this import doesn't follow the same structure as others.

Reply:
This is actually on purpose, we want to import all the contents of the test base so we can access everything. Following the other convention would require a lot more items.